RPIPs/RPIP-43.md

```diff
@@ -22,7 +22,7 @@ This proposal drastically reduces the gas to add validators and distribute rewar
 - A Node Operator SHALL be able to distribute rewards from all validator in their megapool at once
   - This MAY be temporarily blocked while validators are exiting
 - RPL MAY be staked to a megapool
-  - Validators MAY be created within a megapool regardless of RPL staked to the megapool
+  - Validators MAY be created within a megapool regardless of RPL staked to the megapool (allowing ETH-only validators)
```

Comment:
(allowing ETH-only validators) -> (note that this means ETH-only node operation is possible)